stream/stream_file_builder.go

```diff
@@ -0,0 +1,215 @@
+// Authors: Ryan Hollis (ryanh@)
+
+// The purpose of the StreamFile library is to allow streamed writing of XLSX files.
+// It relies heavily on the XLSX library (github.com/tealeg/xlsx).
+// Directions:
+// 1. Create a StreamFileBuilder with NewStreamFileBuilder() or NewStreamFileBuilderForPath().
+// 2. Add the sheets and their first row of data by calling AddSheet().
+// 3. Call Build() to get a StreamFile. Once built, all functions on the builder will return an error.
+// 4. Write to the StreamFile with WriteRow(). Writes begin on the first sheet. New rows are always written and flushed
+// to the io. All rows written to the same sheet must have the same number of cells as the header provided when the sheet
+// was created or an error will be returned.
+// 5. Call NextSheet() to proceed to the next sheet. Once NextSheet() is called, the previous sheet can not be edited.
+// 6. Call Close() to finish.
+
```

Comment:
These directions should be move to (or duplicated in) a new section in the top-level README.org file.  If you're not comfortable with the org-mode formatting let me know and I'll be happy to do that part myself.   Probably we should also indicate that this feature is new and not yet feature complete.